### PR TITLE
 send verify email 

### PR DESCRIPTION
**Pull request by dangthihoa**
related ticket: https://edu-redmine.sun-asterisk.vn/issues/28180
what:  send verify email when users signup success

Screenshot from 2020-08-27 09-53-43
Screenshot from 2020-08-27 09-53-40
Screenshot from 2020-08-27 09-53-14

### CODE REVIEW

**Review by HieuNT-Sun-Asterisk — APPROVED** [review]

**Review by Atanna128 — APPROVED** [review]

**Comment by dangthihoa** [comment]
ready

File: BookingTourSystem/settings.py [hunk 1]
```diff
@@ -149,3 +149,8 @@
         messages.ERROR: 'alert-danger',
  }
 
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'hoadang0107@gmail.com'
```
Comment (thaodtd-0665) [inline BookingTourSystem/settings.py]: dùng biến môi trường nha em

Comment (dangthihoa) [reply BookingTourSystem/settings.py]: vang a

**Comment by dangthihoa** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**